front-end/src/components/navigation-icon/NavigationIcon.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import { withRouter } from 'react-router'
+import SvgIcon from "../svg-icon/SvgIcon";
+import "./NavigationIcon.css"
```

Comment:
I think a better name would fit this css file. And why are we using css if we are compiling scss?